content/it/cloud_native_tech.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Cloud Native Technology
```

Comment:
Please consider translate "Cloud Native Technology" into Italian, if needed!

Ref: "API Gateway" page in pt-br:
- https://github.com/cncf/glossary/pull/479/files
- https://deploy-preview-479--cncfglossary.netlify.app/pt-br/api_gateway/

![image](https://user-images.githubusercontent.com/46767780/155661288-5cd13792-46b0-4887-b492-9c4cc6336ca2.png)

Reply:
Thanks @seokho-son , we agreed keeping this in English makes more sense in Italian!